examples/bottom_bar_bot.py

```diff
@@ -0,0 +1,48 @@
+import logging
+from pyAitu import executor, Bot, Dispatcher
+from pyAitu.models import Message, Options, Form, Header, FormClosed, BottomBar, FormAction, FormSubmitted
+import pyAitu.models.form.input_type as input_type
+import pyAitu.models.form.currency as currency
+
+
+API_TOKEN = 'YOUR_API_TOKEN'
+
+bot = Bot(token=API_TOKEN)
+dispatcher = Dispatcher(bot)
+
+logging.basicConfig(level=logging.INFO)
+
+
+def make_form() -> Form:
+    form_id = "bottom_bar_form_id"
+    header = Header(_type="title", title="Bottom bar component", options=Options(closeable=True))
+    bottom_bar = BottomBar(
+        content_type="bottom_bar",
```

Comment:
update please update your develop branch version, now content_type is given inside of component